controller/contour/daemonset.go

```diff
@@ -267,7 +274,7 @@ func DesiredDaemonSet(contour *operatorv1alpha1.Contour, contourImage, envoyImag
 				{
 					Name:      envoyCfgVolName,
 					MountPath: filepath.Join("/", envoyCfgVolMntDir),
-					ReadOnly:  true,
+					ReadOnly:  false,
```

Comment:
Why switch this to `false`?

Reply:
The init container needs to be able to write out the config.  This logic follows the example here:  https://github.com/projectcontour/contour/blob/main/examples/contour/03-envoy.yaml#L111-L113